### ISSUE TITLE
File loading issue

### ISSUE THREAD

**Issue by n3b0j5a** [THeK3nger/obsidian-chessboard#6]
Since last update I cannot load my chess notes, I got an error that says 
```
"Failed to load "/path/to/chess_note.md". Input does not look algebraic notation.
```
 I also checked in debug console, this is what I see:

![image](https://github.com/THeK3nger/obsidian-chessboard/assets/41470536/34f6e017-b0ee-45f7-857d-0753a96289f1)

So, it might be either compatibility problem with plugin or some bug in Obsidian itself. I'm not familiar with java script so i cannot help here too much.

**Comment by THeK3nger** [comment]
Thank you, I'll take a look. Recently I merged some new annotation features, so maybe something broke there. If you can, an example of broken annotations would be helpful.

**Comment by n3b0j5a** [comment]
Well, probably i have the latest one. It was working fine until last obsidian update. Probably i can clone/test from github directly, idk.

**Comment by THeK3nger** [comment]
No, sorry, I meant: an example of the chessboard block breaking in this version. The content:

````
```chessboard
whatever
```
````

**Comment by n3b0j5a** [comment]
Sure, here are two:

```chessboard
r1bqkbnr/ppp2ppp/2np4/4p3/3PP3/5N2/PPP2PPP/RNBQKB1R
annotations: Af1-b5 Hf1 Ad4-e5 Hd4
```
and:
```chessboard
r1bqkbnr/ppp2ppp/2np4/4p3/3PP3/5N2/PPP2PPP/RNBQKB1Rd
annotations: Af81-b5 Hf8
```
So, nothing complicated.
